src/style.css

```diff
@@ -0,0 +1,100 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir.ttc);
+}
```

Comment:
Follow the checklist, please 
<img width="990" alt="image" src="https://user-images.githubusercontent.com/99403198/216766134-916f81db-5ab0-4264-9b51-403581d7e8ff.png">